vspec/config.yaml

```diff
@@ -215,4 +215,11 @@ units:
     label: kilo newton
     description: Force measured in kilo newton
     domain: force
-  
+  MB:
+    label: megabyte
+    description: Data measured in megabytes
```

Comment:
Regarding the MB/MiB discussion. I think we shall select unit names so we can support both variants (if needed) in the future. And then it could be better to use `MiB` instead of `MB` if that is what we need. Regardless, to avoid misunderstanding  I think it could be good in the description to state what we refer to, by adding e.g. `(base 2)` or `(base 10)` at the end of the description

Reply:
fixed this in the latest commit, thanks for the feedback